teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization, age, name = 'Unknown')
+    super(age, name)
```

Comment:
- [ ] Kindly add parent permission as parameter to initialize and super constructors as its a requirement.

Reply:
Will do, thank you